app/javascript/components/search/results/ResultMetadataTable.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+const maxMetadataEntities = 2
+
+// strip off redundant information like '(disease)' from the end of labels
+// also accounts for array-based labels with pipes |
+function sanitizeMetadataLabel(label) {
+  const allLabels = label.match('|') ? label.split('|') : [label]
+  const returnLabels = []
+  allLabels.map(subLabel => {
+    if (subLabel.endsWith('(disease)')) {
+      returnLabels.push(subLabel.split(' (disease)')[0])
+    } else {
+      returnLabels.push(subLabel)
+    }
+  })
+  return returnLabels.join('|')
+}
+
+// show list of metadata values as badges, truncating as needed
+function metadataList(values, accession, header) {
+  const moreValues = values.splice(maxMetadataEntities)
+  const list = values.map((val, i) => {
+    return <span key={`${accession}-${header}-entry-${i}-val`}
+                 className="label label-primary study-metadata-entry">{sanitizeMetadataLabel(val)}</span>
```

Comment:
```suggestion
                 className="badge badge-secondary study-metadata-entry">{sanitizeMetadataLabel(val)}</span>
```

Applying a visually "primary" style to so many elements is definitely prominent, but also perhaps noisy.  

Would a less prominent style suffice?  Something like `badge badge-secondary` might fit.  The green text and white background would be lower contrast but still very legible.

Its green color would also be consistent with the "metadata matches" badges shown when applying a cross-dataset search filter.  The different _position_ seems sufficient to distinguish that existing "metadata match" badge from these new "metadata available" badges.

Perhaps in later work in another scope, we could further unify these "match" and "available" search result UI elements by e.g. using a more "primary" green badge (green _background_, white _text_ for higher contrast) when an available annotation value is also matched by a selected filter.

I don't feel strongly about that.  If you opt to keep the prominent blue, I'd recommend using `$action-color` (#3D5A87) as the background color.  That's the more SCP-conventional primary blue.

Reply:
I like the idea of visually linking it to the metadata look/feel, so I'm going to go with the inverse styling (green background, white text).  I'm using the same green but slightly transparent (.85) to mute it every so slightly.  Also, I love the idea of making these clickable eventually - that's a really nice enhancement.